src/arch/helperadvsimd.h

```diff
@@ -43,6 +43,14 @@ typedef int32x2_t vint;
 static INLINE int vavailability_i(int name) { return 3; }
 static INLINE void vprefetch_v_p(const void *ptr) { }
 
+static INLINE int vtestallones_i_vo32(vopmask g) {
+  return vget_lane_u32(vpmin_u32(vand_u32(vget_low_u32(g), vget_high_u32(g)), vand_u32(vget_low_u32(g), vget_high_u32(g))), 0);
```

Comment:
I think this could be done more efficiently by doing

```
 uint32x4_t x0 = vpminq_u32(g);
uint32x2_t x1 = vand_u32(vget_low_u32(x0), vget_high_u32(0));
return vget_lane_u32(x1,0);
```